app/controllers/application_controller.rb

```diff
@@ -833,6 +836,14 @@ def add_noindex_header
 
   protected
 
+  def honeypot_value
```

Comment:
Is there a reason we moved the methods to `ApplicationController` instead of `SessionController`?

Reply:
Problem is that this is shared between SessionController and UsersController. In UsersController we check value before account is activated:
```
  def honeypot_or_challenge_fails?(params)
    return false if is_api?
    params[:password_confirmation] != honeypot_value ||
    params[:challenge] != challenge_value.try(:reverse)
  end
```